src/layout/grid/grid.jsx

```diff
@@ -330,6 +338,24 @@ const Grid = React.forwardRef(
             },
         );
 
+        if (responsiveQueryType === 'container') {
+            console.log('pre gridColumns', children);
+        }
```

Comment:
Will this go away when you're done debugging?

Reply:
Remove!